packages/application-components/src/components/detail-pages/tabular-detail-page/tabular-detail-page.spec.tsx

```diff
@@ -85,12 +85,21 @@ describe('rendering', () => {
     screen.getByText(/custom/i);
   });
   it('should warn if no title nor custom title row are provided', () => {
+    const warnSpy = jest.spyOn(console, 'warn').mockImplementation(() => {});
     renderTabularDetailPage();
 
     expect(warning).toHaveBeenCalledWith(
       false,
       'TabularDetailPage: one of either `title` or `customTitleRow` is required but both their values are `undefined`'
     );
+
+    expect(warnSpy).toHaveBeenCalledWith(
+      expect.stringContaining(
+        "Warning: Invalid prop 'title' supplied to 'TextHeadline'. Expected it to be nonempty string, but it was empty."
+      )
+    );
+
+    warnSpy.mockRestore();
```

Comment:
This console.warn call caused multiple tests to fail in CI. It comes from the UI Kit's `TextHeadline` component, which is expected since we're not passing a title. If you'd like me to suppress these warnings without asserting them, let me know.

Reply:
Just wondering if we could provide a value for that prop 🤔

Reply:
But we're testing here `should warn if no title nor custom title row are provided`